process/templates/SIG_NAME_graduation_stage.md

```diff
@@ -1,5 +1,32 @@
 ## Special Interest Group (SIG) graduation
 
 The governing body must agree that the SIG has completed a major deliverable
-
   * Link to relevant documentation.
+
+## SIG has met all Incubating requirements
+  * "link to Incubating PR"
+
+### Governance
+SIG must have documented project governance and be able to demonstrate that governance in action.
+  * "link to governance documents"
+SIG has met at least 4 times over a period of at least 2 months since becoming incubating
+  * "link to meeting agenda"
+Have a define roadmap and annual goals for the project
```

Comment:
These are currently not requirements in the [SIG lifecycle](https://github.com/ossf/tac/blob/main/process/sig-lifecycle.md#to-become-graduated). Since SIGs are meant to be shorter-lived, my suggestion is to make the meeting cadence a requirement for the incubating stage, if at all, and remove the roadmap requirement.

Reply:
```suggestion
```